pyfr/plugins/fluidforce.py

```diff
@@ -117,10 +143,12 @@ def __call__(self, intg):
 
         # Solution matrices indexed by element type
         solns = dict(zip(intg.system.ele_types, intg.soln))
-        ndims, nvars = self.ndims, self.nvars
+        ndims, nvars, mcomp = self.ndims, self.nvars, self._mcomp
 
-        # Force vector
-        f = np.zeros(2*ndims if self._viscous else ndims)
+        # Force and moment vector
+        n = 2 if self._viscous else 1
+        fm = np.zeros(n*(ndims + mcomp))
```

Comment:
We may want to consider using a 2D array here, with n being the second dimension (one or two).  May result in cleaner/more natural code.